resources/scopes/scopes.json

```diff
@@ -0,0 +1,63 @@
+{
+  "scopes": {
+    "android": {
+      "app": null,
+      "browser": null,
+      "engine": null,
+      "resources": null,
+      "samples": null
+    },
+    "ios": {
+      "app": null,
+      "browser": null,
+      "engine": null,
+      "resources": null,
+      "samples": null
+    },
+    "linux": {
+      "config": null,
+      "engine": null,
+      "resources": null,
+      "samples": null
+    },
+    "web": {
+      "bookmarklet": null,
+      "engine": null,
+      "resources": null,
+      "ui": null,
+      "samples": null
+    },
+    "mac": {
+      "config": null,
+      "engine": null,
+      "resources": null,
+      "samples": null
+    },
+    "developer": {
+      "compilers": null,
+      "ide": null,
+      "resources": null,
+      "tools": null
+    },
+    "common": {
+      "core": {
+          "desktop": null,
+          "web": null
+      },
+      "models": {
+        "types": null,
+        "templates": null,
+        "wordbreakers": null
+      },
+      "resources": null,
+      "ci": null
+    },
+    "oem": {
+      "fv": {
+        "android": null,
+        "ios": null,
+        "linux": null
```

Comment:
I thought FV app was for Desktop instead of linux

Reply:
double blush